NUnitTestProject/CalculatorTest.cs

```diff
@@ -0,0 +1,39 @@
+using NUnit.Framework;
+using static NUnitTestProject.Calculator;
+namespace NUnitTestProject
+
+{
+    [TestFixture]
+    public class CalculatorTest
+    {
+        [Test]
+       [Ignore("Ignore a test")]
+        public void ConstructorTest()
```

Comment:
formatting issues
![image](https://github.com/Linases/LinaSaysHello/assets/55654931/2ff9af28-7a9b-4752-9c68-956055483251)


you may use hotkeys ctrl+K (together) + D to format code in VS

Reply:
Fixed